web/libs/editor/src/tags/object/Image/Image.js

```diff
@@ -567,7 +569,11 @@ const Model = types
       };
     },
   }))
-
+  .volatile((self) => {
+    return {
+      manager: null,
+    };
+  })
```

Comment:
we usually use simpler version
```suggestion
  .volatile((self) => ({
    manager: null,
  }))
```